Remotion/ObjectBinding/Web.UnitTests/UI/Controls/BocDateTimeValueImplementation/Rendering/BocDateTimeValueRendererTest.cs

```diff
@@ -267,7 +268,7 @@ private void AssertTime (XmlNode container, BocDateTimeValueRenderer renderer)
         Assert.That(_timeTextBox.Attributes[StubValidationErrorRenderer.ValidationErrorsIDAttribute], Is.EqualTo(c_dateValueID + "_TimeValueValidationErrors"));
         Assert.That(
             _timeTextBox.Attributes[StubValidationErrorRenderer.ValidationErrorsAttribute],
-            Is.EqualTo(s_timeValidationErrors.ToString(WebStringEncoding.Attribute)));
+            Is.EqualTo(s_timeValidationErrors.GetValue()));
```

Comment:
This is just stub-logic anyway, but the implementation of the stub-renderer uses PlainTextString.ToString() for rendering. We shouldn't go with GetValue() here, otherwise this could eventually fail due to encoding mismatch.